server/paymentProviders/stripe/index.js

```diff
@@ -53,6 +53,10 @@ export default {
     default: creditcard,
     creditcard,
     paymentintent,
+    // eslint-disable-next-line camelcase
+    us_bank_account: paymentintent,
+    // eslint-disable-next-line camelcase
+    sepa_debit: paymentintent,
```

Comment:
I like that this is explicit about having different types using the same handler, but at the same time, I'm not sure this is not a manifestation that we don't need to store payment methods at the SEPA/ACH granularity.

Reply:
Agreed. Everything in this provider will be using the same stripe interface, having separate implementations per method would be strange at least. 

I hope we can merge it all at some point and delegate to this provider anything with a PaymentMethod.service == STRIPE. We would still want to store the PaymentMethod.type though, as that is relevant to know if the payment method supports off_session charges, to display to customers saved methods, etc.